### PR TITLE
Bump to version 1.4.3 amd update changelog

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

**Comment by nicoddemus** [comment]
Feel free to hit the merge button once CI passes

**Comment by nicoddemus** [comment]
hey while at it, consider testing with py36 as well 😁

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ language: python
 python: 3.5
```
Comment (nicoddemus) [inline .travis.yml]: You have to specify `python: 3.6` here

Comment (prusse-martin) [reply .travis.yml]: If I do the 3.5 build fails.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 
+matrix:
+  include:
+    - env: TOXENV=py36
+      python: '3.6'
 
+install:
+  - pip install tox
+script:
+- tox
```
Comment (nicoddemus) [inline .travis.yml]: Indent this for consistency please

Comment (prusse-martin) [reply .travis.yml]: Oh... my bad.

**Comment by nicoddemus** [comment]
> Should I add back 3.5 with a similar aproach using matrix and jobs as pointed in the link?

I don't think is needed. LGTM, merging. 👍

**Pull request merged**